src/components/demos/write/LoginForm.js

```diff
@@ -68,56 +74,55 @@ export default function LoginForm({
     }
   }, [usernameInput, passwordInput])
 
+  useEffect(() => {
+    if (failedLogin(apiOutput) || loginError(apiOutput)) {
+      setPasswordInput('')
+      setSomePasswordTyped(false)
+    }
+  }, [apiOutput])
+
   return (
-    <form onSubmit={enterUsernameAndPassword}>
+    <Form onSubmit={enterUsernameAndPassword}>
       <Input
         label={'Username'}
         theme={demoState.theme.form}
         setInputState={setUsernameInput}
         showInputError={usernameAtMaxLength || noUsernameEntered}
         InputErrorWarning={UsernameErrorWarning}
         cySelector={'username'}
+        style={{ alignSelf: 'center' }}
       />
       <Input
         label={'Password'}
         theme={demoState.theme.form}
+        value={passwordInput}
         setInputState={setPasswordInput}
         upperBound={passwordUpperBound}
         inputType={'password'}
         showInputError={passwordAtMaxLength || noPasswordEntered}
         InputErrorWarning={PasswordErrorWarning}
         cySelector={'password'}
+        style={{ alignSelf: 'center' }}
       />
-      <Container
```

Comment:
removed this container as it was causing some DOM dancing when the login failed text showed. The increased font size means the text is wider than anything else in the form and it was moving things around when it appeared.